Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -306,7 +308,7 @@ class Detector : public o2::Base::Detector
     GeometryHandler *mGeometryHandler;
     MisalignmentParameter *mMisalignmentParameter;
 
-    V1Layer **mGeometry;   //! Geometry
+    V3Layer **mGeometry;   //! Geometry
```

Comment:
Probably you want to express something like array of V3Layers. So, I am wondering if we could use something like `std::vector<V3Layer*>` or `std::vector<unique_ptr<V3Layer>>` or similar. We can possibly address this in a separate improvement PR.